packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -513,7 +513,7 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
       return { xBarScale, yBarScale };
     } else {
       const xBarScale = d3ScaleBand()
-        .domain(this._xAxisLabels)
+        .domain(this._isRtl ? this._xAxisLabels.slice().reverse() : this._xAxisLabels)
```

Comment:
domain is not changing, range is changing.
Make change to update the range if rtl

Reply:
yes, fixed it